sources/Common/src/Options.cpp

```diff
@@ -154,37 +123,13 @@ Options::parse(int argc, char* argv[]) {
     } else {
       return Request::RUN_SIMULATION_N;
     }
-  } catch (const zip::ZipException& e) {
+  } catch (const std::exception &e) {
     std::cerr << e.what() << std::endl;
     return Request::ERROR;
-  } catch (const std::exception& e) {
-    std::cerr << e.what() << std::endl;
-    return Request::ERROR;
-  }
-}
-
-bool
-Options::areRequiredFilesMissing(const po::variables_map& vm, const boost::shared_ptr<zip::ZipFile>& archive) const {
```

Comment:
So we totally abandon this check ?

Reply:
it is done later in the code when trying to load the files